src/libraries/Native/Windows/CMakeLists.txt

```diff
@@ -65,6 +65,13 @@ if (${CLR_CMAKE_HOST_ARCH} STREQUAL "x86")
     add_compile_options(/Gz)
 endif ()
 
+if(${CLR_CMAKE_HOST_ARCH} STREQUAL arm OR ${CLR_CMAKE_HOST_ARCH} STREQUAL arm64)
+    add_compile_options(-DZLIB_COMPAT=0) # Must be explicitly set
```

Comment:
We don't want a zlib compatible API? What does this actually affect?

Reply:
It affects the naming of the functions and I believe some function argument types.

Reply:
https://github.com/zlib-ng/zlib-ng/blob/develop/PORTING.md